app/views/users/index.html.erb

```diff
@@ -1,2 +1,8 @@
-<h1>Here is a list of all the users</h1>
+<main>
```

Comment:
- [ ] Your application is displaying a blank screen, and hence not working as intended. Kindly ensure that your app is properly routed to the root.

Reply:
Hi @rloterh, the reason why it is displaying only a blank screen is that the database does not have data, as specified in the project requirements, the person using it/testing should add data to the database (at least in the current state) due to the app not having a way to add new data to its database (not required by the project requirements).

What the project requirements say:
"Hint: In order to see some posts or comments, you might need to add some data to your database as described [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data)."

Reply:
If there is no resolution in 45 minutes I will be requesting a second opinion.

Reply:
Hi @KatIsCoding, for this module `Views` your app is required to function as [these wireframes ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md) as a result you need to populate your database to be able to achieve this. 

> Hi @rloterh, the reason why it is displaying only a blank screen is that the database does not have data, as specified in the project requirements, the person using it/testing should add data to the database (at least in the current state) due to the app not having a way to add new data to its database (not required by the project requirements, at least not until now).
> 
> What the project requirements say: "Hint: In order to see some posts or comments, you might need to add some data to your database as described [here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data)."

The link you are referring to here is for the student and not visitors of your repository, kindly populate your database so that the methods inside your models and views would function as they should [You can checkout this article on how to use Seed.rb file to solve this](https://ninjadevel.com/seeding-database-ruby-on-rails/)
![image](https://user-images.githubusercontent.com/52497246/153353562-49d17c0f-1955-402e-ae92-98b5d01768d9.png)

Reply:
Hello @Davidosky007 
Thanks very much for throwing more light on this

Reply:
Hi @KatIsCoding 
I missed the steps of adding data via the console during my review. I apologize for missing this important step. Kindly know that this was not a deliberate attempt.

Cheers, and happy coding! ☺️